sdk/servicebus/azure-servicebus/tests/test_subscriptions.py

```diff
@@ -156,7 +156,7 @@ def test_subscription_by_servicebus_client_receive_batch_with_deadletter(self, s
                 count = 0
                 for message in receiver:
                     print_message(_logger, message)
-                    message.complete()
+                    receiver.message.complete_message(message)
```

Comment:
receiver.message?

Reply:
aha, I know why I miss this part: this line actually doesn't get executed as the logic is to verify that no message is received.
thanks for your eyes.